globex-questions.md

```diff
@@ -44,8 +44,8 @@ Note: You should have already followed the [set-up instructions](https://github.
 2.  How many units of the product from the previous question were sold on the credit card with number `3535880159004410`?
     <details><summary>Answer</summary>7098.00</details>
 
-3.  We've noticed an error in an upstream system. When updating an employee's manager the the employees table hasn't been correctly updated. The `ManagerId` has been updated, but the `ManagerFirstName` and `ManagerSurname` have not. How many records does this impact?
-    <details><summary>Answer</summary>3</details>
+3.  We've noticed an error in an upstream system. When updating an employee's manager the employees table hasn't been correctly updated. The `ManagerId` has been updated, but the `ManagerFirstName` and `ManagerSurname` have not. How many records does this impact?
+    <details><summary>Answer</summary>4</details>
```

Comment:
This was 3 if only comparing first names, but 4 when accounting for surnames too